store/file_store_test.go

```diff
@@ -182,6 +182,22 @@ func TestCreate(t *testing.T) {
 	if exists(root, scratchdir, "abc") {
 		t.Errorf("File scratch abc exists")
 	}
+
+	_, err = s.Create("abc/abc")
+	if err != ErrKeyContainsSlash {
+		t.Errorf("Received error %s", err.Error())
+	}
+
+	_, err = s.Create("def abc")
+	if err != ErrKeyContainsWhiteSpace {
+		t.Errorf("Received error %s", err.Error())
+	}
+
+	_, err = s.Create("defabc")
```

Comment:
Is this an embedded control character?! Props to you. Do you think using a hex escape would be clearer? e.g. `\x0c`.